jobProcess.ts

```diff
@@ -1,22 +1,23 @@
-import { git, elastic } from './connect/queue'
+import { git, elastic, telegramSendUser } from './connect/queue'
 import payload from './utils/payload'
 import connectElastic from './connect/elastic'
 import delay from './utils/delay'
 import templateBody from './template/body'
 import captureException from './capture/exception'
 import sendTelegram from './send/telegram'
 import { sendBodyIsNotValid } from './send/elastic'
+import { sendMessageWithUser } from './utils/telegram'
 
 git.process(async (job, done) => {
   delay()
   const data = payload(job.data)
   try {
     const body = await templateBody(data)
-    console.log('start');
+    console.log('start')
     sendTelegram(body)
       .then(() => done())
       .catch(error => {
-        console.log('error');
+        console.log('error')
```

Comment:
just ask, why not write to file instead print on console for error or info?

Reply:
log full error on sentry